projects/components/src/overlay/sheet/sheet-overlay.component.scss

```diff
@@ -20,13 +20,19 @@
   &.sheet-size-small {
     width: 320px;
   }
+
   &.sheet-size-medium {
     width: 600px;
   }
+
   &.sheet-size-large {
     width: 840px;
   }
 
+  &.sheet-size-extra-large {
```

Comment:
Isn't this bigger than the widget we even design for?

Reply:
Hah.. Few things are in flux. I spoke to Michael about it. Yes, It looks too big for laptop screen. 

There would be some more changes around this as he needs more time to play with it. Happy to revert this later.